### PR TITLE
swashbuckle-nodatime Issue 1: Support for nullable types

### PR DESCRIPTION
**Pull request by SteveK-GS**

### CODE REVIEW

File: src/Swashbuckle.NodaTime/SwaggerDocsConfigExtensions.fs [hunk 1]
```diff
@@ -32,6 +32,16 @@ module SwaggerDocsConfigExtensions =
...
 
+        config.MapType<System.Nullable<Instant>>(fun () -> schemas.Instant)
+        config.MapType<System.Nullable<LocalDate>>(fun () -> schemas.LocalDate)
+        config.MapType<System.Nullable<LocalTime>>(fun () -> schemas.LocalTime)
+        config.MapType<System.Nullable<LocalDateTime>>(fun () -> schemas.LocalDateTime)
+        config.MapType<System.Nullable<OffsetDateTime>>(fun () -> schemas.OffsetDateTime)
+        config.MapType<System.Nullable<ZonedDateTime>>(fun () -> schemas.ZonedDateTime)
+        config.MapType<System.Nullable<Interval>>(fun () -> schemas.Interval)
+        config.MapType<System.Nullable<Offset>>(fun () -> schemas.Offset)
+        config.MapType<System.Nullable<Duration>>(fun () -> schemas.Duration)
```
Comment (jakubka) [inline src/Swashbuckle.NodaTime/SwaggerDocsConfigExtensions.fs]: I noticed that nullable `Period` and `DateTimeZone` aren't configured. Did you miss those two or there is a reason for that?

**Comment by jakubka** [comment]
Thanks for the contribution @SteveK-GS! I added one question.

Could you also please bump a version in AssemblyInfo to `2.1.0` please as this is a new feature? 

In the meantime, I will integrate this project with CI so that it's pushed to Nuget automatically.

**Comment by SteveK-GS** [comment]
Thank you @jakubka. I don't recall any good reason for missing those two - I will look into it and also increment the assembly version.

**Comment by SteveK-GS** [comment]
So, @jakubka it turns out there was a good reason to omit the 2 types you highlighted - they are not structs so are nullable out of the box. A null _Period_, for example, is of type _Period_, whereas a null _Duration_ is a different type altogether - _System.Nullable<Duration>_.

**Comment by SteveK-GS** [comment]
I'm still learning Git and GitHub and it appears that I should have created a branch after forking so that I can if need be work on other issues at the same time (namely #3 ). I've done that retrospectively and created a new PR for #1  from the new branch ( #4 ). Please accept my apologies if I messed up.

**Pull request closed without merging**